### ISSUE TITLE
ImageNet dataset

### ISSUE THREAD

**Issue by qinghezeng** [deu30303/RUC#13]
Thank you so much for your amazing work!

Could you please provide the script used for ImageNet dataset? 

I would like to train RUC on my own dataset which has a similar image size to ImageNet. However, I cannot train the SCAN-pretrained model (MOCO) with an 8G GPU, even if I resize the image to 96x96 and set the batch size to 2. Thus I would like to ask, which GPUs did you use in your ImageNet experiments and how much memory do they have? If I understand your paper well, you resize them to 256x256, right? And what is the batch size you use? 

Many thanks in advance!

**Comment by deu30303** [comment]
We used 6 A100 GPU to train imagenet dataset (256 X 256). 
We used the pretrained clustering model from SCAN github and batch size was 16~32 

Thank you again for your interest.

**Comment by qinghezeng** [comment]
Thanks for your quick answer! 

Could you please share your settings for ImageNet training? @deu30303 For example, learning rate, epochs, etc.

Another question, I notice that in the function `criterion_rb`, a different value is used for N
`Lu = N * torch.mean((probs_u - targets_u)**2) `
(stl10 N=50, cifar10 N=25, cifar20 N=100). How shall I choose the value for my own dataset?

Thank you very much in advance!

**Comment by deu30303** [comment]
For ImageNet, opimizer : SGD,  Learning rate : 0.01 , Weight decay: 5e-4, Epoch : 100, lambda_u : 25
For CIFAR and STL dataset, our model is not sensitive to the lambda_u hyperparameter. 
(Please refer table 7 in our paper [link](https://arxiv.org/pdf/2012.11150.pdf))

Thank you again for your interest.
